doc/en/example/parametrize.rst

```diff
@@ -127,11 +127,21 @@ objects, they are still using the default pytest representation::
 
     $ py.test test_time.py --collect-only
     ======= test session starts ========
-    platform linux2 -- Python 2.7.9, pytest-2.8.0.dev4, py-1.4.28, pluggy-0.3.0
+    platform linux -- Python 3.4.2, pytest-2.8.1.dev1, py-1.4.30, pluggy-0.3.1
     rootdir: $REGENDOC_TMPDIR, inifile: 
+    collected 0 items / 1 errors
     
-    =======  in 0.12 seconds ========
-    ERROR: file not found: test_time.py
+    ======= ERRORS ========
+    _______ ERROR collecting test_time.py ________
+    $PYTHON_PREFIX/lib/python3.4/site-packages/_pytest/python.py:581: in _importtestmodule
+        mod = self.fspath.pyimport(ensuresyspath=importmode)
+    $PYTHON_PREFIX/lib/python3.4/site-packages/py/_path/local.py:650: in pyimport
+        __import__(modname)
+    E     File "$REGENDOC_TMPDIR/test_time.py", line 5
+    E       (datetime(2001, 12, 11), datetime(2001, 12, 12), timedelta(-1)),
+    E                                                                      ^
+    E   SyntaxError: unexpected EOF while parsing
+    ======= 1 error in 0.12 seconds ========
```

Comment:
Why does this happen? While it was broken before this PR already, I don't see what's broken now - the source above seems to run fine for me?